### PR TITLE
Adds the ability to control idle timeout action

### LINKED ISSUE #229: IIS 8.5 Idle Timeout Action support
In IIS 8.5 and above, Idle Timeout Action is introduced.
http://www.iis.net/learn/get-started/whats-new-in-iis-85/idle-worker-process-page-out-in-iis85
Default is Terminate, but Suspend is an option that allows idle app pool to wake up quickly, eliminating long load time.
Can we add this feature into the cookbook?

### PR DESCRIPTION
**Pull request by EasyAsABC123**
Resolves #229

### CODE REVIEW

File: resources/pool.rb [hunk 1]
```diff
@@ -43,6 +43,7 @@
 attribute :logon_type, kind_of: Symbol, equal_to: [:LogonBatch, :LogonService], default: :LogonBatch
 attribute :manual_group_membership, kind_of: [TrueClass, FalseClass], default: false
 attribute :idle_timeout, kind_of: String, default: '00:20:00'
+attribute :idle_timeout_action, kind_of: Symbol, equal_to: [:Terminate, :Suspend]
```
Comment (calebih) [inline resources/pool.rb]: Default is :Terminate. This should be added here.

**Comment by calebih** [comment]
looks good.

**Comment by EasyAsABC123** [comment]
@calebih were you able to run a recipe testing it? I didn't have time yet

**Comment by calebih** [comment]
Yes. I tested it in my environment, and I was able to set IdleTimeoutAction through cookbook. Thanks.

**Pull request merged**